torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -907,16 +908,16 @@ def call_function(
         # We compute the example_value by actually calling
         # `vmap` with FakeTensors.
         fake_batched_fn_args = tuple(
-            arg.as_proxy().node.meta["example_value"] for arg in batch_input_args
-        ) + tuple(arg.node.meta["example_value"] for arg in body_lifted_freevars)
```

Comment:
slight nit: You could itertools.chain() these two generators together so you don't need a tuple concatenation.